oak/server/oak_node.cc

```diff
@@ -345,28 +323,42 @@ wabt::interp::HostFunc::Callback OakNode::OakWaitOnChannels(wabt::interp::Enviro
     results[0].set_i32(OakStatus::OK);
 
     if (count == 0) {
-      LOG(INFO) << "Waiting on no channels";
+      LOG(INFO) << "Waiting on no channels, return immediately";
       return wabt::interp::Result::Ok;
     }
 
-    uint64_t handle0 = ReadU64(env, offset);
-    // TODO: Drop hardcoded single channel
-    if (handle0 != ChannelHandle::GRPC_IN) {
-      LOG(ERROR) << "Read of unexpected handle " << handle0;
-      return wabt::interp::Result::Ok;
+    std::vector<uint64_t> handles;
+    handles.reserve(count);
+    for (uint32_t ii = 0; ii < count; ii++) {
```

Comment:
What's wrong with "i"? :)

Reply:
I'm just going to leave this here https://www.reddit.com/r/ProgrammerHumor/comments/27uqmp/emojis_as_functionvariable_names_it_works/ . cc @anghelcovici

Reply:
ii is twice as good as i 🙃